src/Telnyx.net/Entities/Messaging/Messaging_Sender_Ids/MessagingSenderId.cs

```diff
@@ -4,6 +4,54 @@ namespace Telnyx
     using System.Runtime.Serialization;
     using Newtonsoft.Json;
 
+    public class MessagingFrom
```

Comment:
`"to": [
      {
        "carrier": "T-MOBILE USA, INC.",
        "line_type": "Wireless",
        "phone_number": "+18665550001",
        "status": "queued"
      }
    ]`
Might be an array of these objects according to the docs

Reply:
Docs are wrong I think